lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -678,24 +685,24 @@ class LinearMapInfo {
       astFnTy = FunctionType::get(
           params, silFnTy->getAllResultsInterfaceType().getASTType());
 
-    auto *origBB = inst->getParent();
+    auto *origBB = ai->getParent();
     auto *linMapStruct = getLinearMapStruct(origBB);
     std::string linearMapName;
     switch (kind) {
     case AutoDiffLinearMapKind::Differential:
-      linearMapName = "differential_" + llvm::itostr(linearMapValueMap.size());
+      linearMapName = "differential_" + llvm::itostr(linearMapFieldMap.size());
       break;
     case AutoDiffLinearMapKind::Pullback:
-      linearMapName = "pullback_" + llvm::itostr(linearMapValueMap.size());
+      linearMapName = "pullback_" + llvm::itostr(linearMapFieldMap.size());
       break;
     }
     auto *linearMapDecl = addVarDecl(linMapStruct, linearMapName, astFnTy);
-    linearMapValueMap.insert({inst, linearMapDecl});
+    linearMapFieldMap.insert({ai, linearMapDecl});
     return linearMapDecl;
   }
 
-  /// Given an `apply` instruction, conditionally adds its linear map function
-  /// to the linear map struct if it is active.
+  /// Given an `apply` instruction, conditionally add a linear map struct field
```

Comment:
This comments uses `add`, but the comment on `lookUpLinearMapDecl` below uses `Finds`. We need consistency.

Reply:
Changed to `adds` for consistency in https://github.com/apple/swift/pull/28251/commits/32640e1fcd4d0938e229a1532288f90a969b41ff.